### PR TITLE
[READY] Optimize the Workshop landing page

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Add Schema.org Event markup for SEO purposes
- [x] Optimize content (textual)
- [x] Add sharing options 
- [x] Layout fixes (conversion rate optimization)
- [x] some enters for readability
- [x] a more specific URL (e.g. /workshop-convenant-mt or /workshop-convenant-medische-technologie) 
- [x] Update of CTA's anchor text

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
@sn3p ready to review, except the 301 redirect

File: source/workshop-convenant-mt.html.erb [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+      <p><em>Kun je niet bij de workshop zijn? Geef je dan toch op. We zullen alle sheets en handouts naar je sturen</em></p>
+    </article>
+  </div>
+</section>
+
+<section class="program gray">
+  <div class="row">
+    <h3>Het programma</h3>
+    <div class="row program-1">
+    <ul class="program-list">
```
Comment (sn3p) [inline source/workshop-convenant-mt.html.erb]: Try to use proper indentation in the future. Clean code is readable code ;)
Same goes for CSS (I just committed some changes)

**Comment by sn3p** [comment]
Looks good :+1:

File: source/workshop-convenant-mt.html.erb (hunk 1 above)
Comment (fatboypunk) [inline source/workshop-convenant-mt.html.erb]: but you didn't commit this file? o.O

**Pull request merged**